.circleci/config.yml

```diff
@@ -0,0 +1,63 @@
+# Ruby CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-ruby/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+      # specify the version you desire here
+      - image: circleci/ruby:2.4.1-node-browsers
```

Comment:
1. Is ruby 2.4.1 the version you are using? If not, update it here to match.
1. Because you perform no browser check here, you can use the [version without `-browsers`](app/scripts/features/mimo/emailTemplates/components/form.jsx).